raiden-dapp/src/components/AppCore.vue

```diff
@@ -1,13 +1,25 @@
 <template>
-  <tokens />
+  <NoTokens v-if="!tokens.length" />
```

Comment:
```suggestion
  <no-tokens v-if="!tokens.length" />
```

In all the components we use the components in kebab case.
We should also add a lint rule to enforce this.

Reply:
Done